js/config.js

```diff
@@ -4,29 +4,33 @@ var respecConfig = {
         "uri" : "Digikoppeling-Best-Practices-GB.pdf"
       } ],
   authors: [ { 
-        "mailto" : "digikoppeling@logius.nl",
         "name" : "Logius",
         "url" : "https://logius.nl/standaarden"
       } ],
   editors: [ 
       { 
         "company" : "Logius",
         "name" : "Peter Haasnoot",
-        "url" : "https://logius.nl/standaarden"
+        "companyURL" : "https://logius.nl/standaarden"
       },
       { 
         "company" : "Logius",
         "name" : "Pieter Hering",
-        "url" : "https://logius.nl/standaarden"
+        "companyURL" : "https://logius.nl/standaarden"
+      },
+      { 
+        "company" : "Logius",
+        "name" : "Martin van der Plas",
+        "companyURL" : "https://logius.nl/standaarden"
       }
     ],
   github: "https://github.com/Logius-standaarden/Digikoppeling-Best-Practices-GB",
   previousPublishDate: "2017-10-01",
-  previousPublishVersion: "3.1",
+  previousPublishVersion: "3.1.1",
   pubDomain: "dk",
-  publishDate: "2022-04-11",
-  publishVersion: "3.1.1",
+  publishDate: "2023-05-16",
+  publishVersion: "3.2",
```

Comment:
```suggestion
  publishVersion: "3.2.0",
```
Bij "Documentbeheer" staat `3.2.0`.
https://github.com/Logius-standaarden/Digikoppeling-Best-Practices-GB/blob/90e8e9b99dbe66fd4804e033702dbb1aecb54be2/02_documentbeheer.md?plain=1#L10